packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point-view.tsx

```diff
@@ -101,10 +101,14 @@ export const MovablePointView = forwardRef(
                 tabIndex={
                     disableKeyboardInteraction ? -1 : tabIndex(focusBehavior)
                 }
-                onFocus={() => {
-                    return getOnFocusChangeCallback(focusBehavior)(true);
+                onFocus={(event) => {
+                    const callback = getOnFocusChangeCallback(focusBehavior);
+                    callback && callback(event, true);
+                }}
+                onBlur={(event) => {
+                    const callback = getOnFocusChangeCallback(focusBehavior);
+                    callback && callback(event, false);
```

Comment:
Do you need to check for the existence of the callback before calling it? Seems like it will always be defined.

Reply:
<img width="1335" alt="image" src="https://github.com/user-attachments/assets/9ea64fc5-9c85-4a45-b4bf-590e2969822c">

Sooooo yes! I totally agree but typescript doesn't.